package.json

```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "@navikt/boxed-list-with-links": "^1.0.2",
-    "@navikt/familie-backend": "4.1.3",
+    "@navikt/familie-backend": "^4.1.3",
```

Comment:
Have a look at the edited ci-screenshots. It does not look right. It will probably change when rewriting the package I guess.

Reply:
fixed